src/applications/personalization/profile-2/components/personal-information/EmailInformationSection.jsx

```diff
@@ -43,10 +43,10 @@ const EmailInformationSection = ({ className, signInServiceName }) => {
                 </p>
                 <p>
                   To update the email you use to sign in, go to the website
-                  where you manage your log in information.
+                  where you manage your sign in information.
```

Comment:
@tressaellen @peggygannon  we use `sign in` here but `sign-in` below...

Reply:
Checked with content - both should be **sign in** , not sign-in. (sans - )